packagelist/ubuntu_23.04_x11.txt

```diff
@@ -78,6 +78,8 @@ libubootenv-tool
 lsof
 meson
 nfs-common
+sqlite3
+xxd
```

Comment:
Make sure any tools you're adding for Debian also get added to the Ubuntu lists so these images are mostly consistent.

Reply:
Right, thanks!